src/main/java/org/apache/flink/state/benchmark/ListStateBenchmarkBase.java

```diff
@@ -79,19 +87,37 @@ public void tearDown() {
 
     @Benchmark
     public void test1Update() throws Exception {
-        keyedStateBackend.setCurrentKey(random.nextLong(setupKeyCount));
-        listState.update(Collections.singletonList(random.nextLong()));
+        int currentIndex = getCurrentIndex();
+        keyedStateBackend.setCurrentKey(setupKeys.get(currentIndex % setupKeyCount));
+        listState.update(Collections.singletonList(randomValues.get(currentIndex % randomValueCount)));
```

Comment:
here you are also benchmarking the creation of the list, maybe consider creating the list in `@Setup` context and benchmarking only `setCurrentKey()` and `update()`?

(the same applies to other benchmarks)